src/JWT.Extensions.AspNetCore/JwtAuthenticationEvents.cs

```diff
@@ -0,0 +1,99 @@
+namespace JWT.Extensions.AspNetCore;
```

Comment:
not: for consistency with the rest of the codebase, please don't use implicit namespace